app/src/main/resources/com/oracle/javafx/scenebuilder/app/i18n/SceneBuilderApp.properties

```diff
@@ -511,6 +511,9 @@ welcome.recent.items.loading = Loading recent projects...
 welcome.recent.items.no.recent.items = no recent projects
 welcome.open.project.label = Open Project
 welcome.loading.label = Loading Components...
+welcome.loading.when.dropped.error.title=Unsupported file format or empty directory
+welcome.loading.when.dropped.error.message=The dropped object is either not a JavaFX FXML file or does not contain any FXML files to be loaded.
```

Comment:
I would keep it simple, similar to that of `alert.open.failureMofN.details`: 

> Unable to load some dropped file(s). Make sure file(s) are valid FXML documents.

Reply:
I've reworked the window. The `MofN` approach does not seem suitable to me due to the usage pattern as described below. It looks now as follows:

![grafik](https://github.com/user-attachments/assets/7b279289-1da7-48c2-bbb4-3e4c0ce074c9)

The text stops after 5 items. If one wants to see all details, one must click on `Show details`. The message itself is now somewhat simpler.

![grafik](https://github.com/user-attachments/assets/d5438e6b-8033-485c-bb45-3b8d8995167e)

What do you think?

Reply:
Alternatively one could display a message like:

![grafik](https://github.com/user-attachments/assets/014571e2-5d8f-4b4d-b889-60bca3a2a66c)

Or:

![grafik](https://github.com/user-attachments/assets/647622a4-2a5f-44c9-8821-93aba06efb7c)

Or (my preference):

![grafik](https://github.com/user-attachments/assets/38ea9979-b9ea-49f4-af8e-38511a8a86e8)

Reply:
Looks good. I would not show the directory path here.

Reply:
![grafik](https://github.com/user-attachments/assets/dec76275-ce47-4296-bd96-6dd6050af7a0)

I'll keep the list of files in the details view but remove them from the actual dialog.

Reply:
Its now reworked accordingly.